most.js

```diff
@@ -23,6 +23,24 @@ exports.from     = from;
 exports.periodic = periodic;
 
 //-----------------------------------------------------------------------
+// Fluent adapter
+
+var thru = require('./lib/combinator/thru').thru;
+
+exports.thru = thru;
```

Comment:
I dunno about this ... the whole point is to use thru as a method.  Having it as a function doesn't make much sense.  I'll remove it and if someone sees a reason it should exist, we can easily add it back.